ui/components/MesheryFilters/FiltersGrid.js

```diff
@@ -31,7 +31,8 @@ function FilterCardGridItem({ filter, handleDeploy, handleUndeploy, handleSubmit
         handleDeploy={handleDeploy}
         handleUndeploy={handleUndeploy}
         handleClone={handleClone}
-        deleteHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.DELETE ,name : filter.name })}
+        deleteHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.DELETE, name : filter.name })}
+        updateHandler={() => handleSubmit({ data : yaml, id : filter.id, type : FILE_OPS.UPDATE ,name : filter.name })}
```

Comment:
Has the data: yaml been remapped to the new Component / Config field?

Reply:
Yes, I've mapped it now.

Reply:
Here: https://github.com/meshery/meshery/pull/7992/files#diff-6ec8e150ffa5db5b0830cd1598d529bbc98c79c98df32092858a51a779fcdea0R693